tests/test_basic.py

```diff
@@ -219,14 +221,14 @@ def my_function2(n):
 
             def t1():
                 a = subprocess.run(
-                    ["ls result_my_function2*.json"], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+                    ["ls result_my_function2*.json"], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
```

Comment:
I don't like the comma here, let's do something like

```python
a = subprocess.run(["ls result_my_function2*.json"],
                   shell=True,
                   stdout=subprocess.PIPE,
                   stderr=subprocess.PIPE)
```

Same for the code below.